### ISSUE TITLE
test: test/test-connect.js fails

### ISSUE THREAD

**Issue by bnoordhuis** [bnoordhuis/node-unix-dgram#24]
At commit 17197be:

```
$ iojs test/test-connect.js
connected
client send PING0
client send PING1
client send PING2
client send PING3
client send PING4
client send PING5
client send PING6
client send PING7
client send PING8
client send PING9
client send PING10
congestion
server recv PING0 { '0': <Buffer 50 49 4e 47 30>,
  '1': { size: 5, address: {}, path: null } }
server recv PING1 { '0': <Buffer 50 49 4e 47 31>,
  '1': { size: 5, address: {}, path: null } }
writable
client send PING11
server recv PING2 { '0': <Buffer 50 49 4e 47 32>,
  '1': { size: 5, address: {}, path: null } }
writable
client send PING12
server recv PING3 { '0': <Buffer 50 49 4e 47 33>,
  '1': { size: 5, address: {}, path: null } }
writable
client send PING13
server recv PING4 { '0': <Buffer 50 49 4e 47 34>,
  '1': { size: 5, address: {}, path: null } }
writable
client send PING14
server recv PING5 { '0': <Buffer 50 49 4e 47 35>,
  '1': { size: 5, address: {}, path: null } }
writable
client send PING15
server recv PING6 { '0': <Buffer 50 49 4e 47 36>,
  '1': { size: 5, address: {}, path: null } }
writable
client send PING16
server recv PING7 { '0': <Buffer 50 49 4e 47 37>,
  '1': { size: 5, address: {}, path: null } }
writable
client send PING17
server recv PING8 { '0': <Buffer 50 49 4e 47 38>,
  '1': { size: 5, address: {}, path: null } }
writable
client send PING18
server recv PING9 { '0': <Buffer 50 49 4e 47 39>,
  '1': { size: 5, address: {}, path: null } }
writable
client send PING19
server recv PING10 { '0': <Buffer 50 49 4e 47 31 30>,
  '1': { size: 6, address: {}, path: null } }
writable
client send PING20
server recv PING11 { '0': <Buffer 50 49 4e 47 31 31>,
  '1': { size: 6, address: {}, path: null } }

assert.js:88
  throw new assert.AssertionError({
        ^
AssertionError: 12 == 21
    at process.<anonymous> (/home/bnoordhuis/src/node-unix-dgram/test/test-connect.js:11:10)
    at emitOne (events.js:77:13)
    at process.emit (events.js:166:7)
```

/cc @santigimeno

**Comment by santigimeno** [comment]
@bnoordhuis Fixed in https://github.com/bnoordhuis/node-unix-dgram/pull/25

**Comment by bnoordhuis** [comment]
Fixed in f5a7309 and mea culpa.
